source/common/access_log/grpc_access_log_impl.cc

```diff
@@ -131,6 +131,12 @@ void HttpGrpcAccessLog::responseFlagsToAccessLogResponseFlags(
   if (request_info.getResponseFlag(RequestInfo::ResponseFlag::RateLimited)) {
     common_access_log.mutable_response_flags()->set_rate_limited(true);
   }
+
+  // TODO(saumoh): Uncomment once unauthorization has been added to accesslog.proto
+  //
+  // if (request_info.getResponseFlag(RequestInfo::ResponseFlag::Unauthorized)) {
```

Comment:
I would remove all of this for now and keep it locally. It will bit rot otherwise.